src/lib/components/Header/Header.svelte

```diff
@@ -39,17 +39,25 @@
         <Icon src={closeIcon} alt="Close" size={3} />
       </button>
 
+      <a class="ldaf-logo__compact" href="/" title="<Project title>">
+        <img src={ldafLogo} alt="LDAF Logo" />
+      </a>
+      <div class="ldaf-commissioner__compact">
+        <span>{siteTitle.commissionerRow1}</span>
+        <span class="text-italic">{siteTitle.commissionerRow2}</span>
+      </div>
+
       <Nav items={navItems} />
 
       <!-- TODO: Extend <Nav/> to cover secondary nav or build out component with shared dependencies. -->
-      <div class="usa-nav__secondary">
+      <div class="ldaf-nav__secondary usa-nav__secondary">
         <ul class="usa-nav__secondary-links">
-          <li class="usa-nav__secondary-item">
-            <a href="/news">Grants and funding</a>
-          </li>
-          <li class="usa-nav__secondary-item">
-            <a href="/documentation" class="usa-current">Licensing and permits</a>
-          </li>
+          {#each secondaryNavItems as item, i (item.id)}
+            {@const { name, link } = item}
+            <li class="usa-nav__secondary-item">
+              <a href={link}>{name}</a>
```

Comment:
We should use `Link` here instead, mainly because in the future I expect we'll probably be adding some sort of analytics events to that component that wouldn't get captured if we just used an `<a>`.

You could also set `alternate={true}` to get the right color by default on desktop, but this might not be the right call since we're styling mobile-first and would still need to override the color on mobile.